src/app/hooks/useClickTrackerHandler/index.jsx

```diff
@@ -157,12 +159,18 @@ export default (eventTrackingData = {}) => {
   });
 
   const enableStaticTracking = !isHydrated && !isAmp;
+  const reverbStaticUrl = constructReverbUrl({
+    eventTrackingData,
+    eventType: CLICK_EVENT,
+  });
 
   return {
     ...(enableStaticTracking && {
       [STATIC_ATI_CLICK_TRACKING]: staticAtiUrl,
     }),
-
+    ...(enableStaticTracking && {
```

Comment:
Can we remove `STATIC_ATI_CLICK_TRACKING` now that we have `STATIC_REVERB_CLICK_TRACKING`? 

```suggestion
    ...(enableStaticTracking && {
```

Reply:
Yesss, definitely.